Buildings/Media/Interfaces/PartialPureSubstanceWithSat.mo

```diff
@@ -0,0 +1,92 @@
+within Buildings.Media.Interfaces;
+partial package PartialPureSubstanceWithSat
```

Comment:
Should the name be "PartialPureSubstanceWithSaturation"?

Reply:
Yes, revised as suggested.